crates/task-impls/src/quorum_proposal.rs

```diff
@@ -102,6 +108,148 @@ struct ProposalDependencyHandle<TYPES: NodeType> {
     id: u64,
 }
 
+impl<TYPES: NodeType> ProposalDependencyHandle<TYPES> {
+    /// Sends a proposal if possible from the high qc we have
+    #[allow(clippy::too_many_lines)]
+    #[allow(dead_code)]
+    pub async fn publish_proposal_if_able(
```

Comment:
How come we added this here instead of just calling the function in helpers?

Reply:
Looks like `publish_proposal_if_able` in `helpers` is imported in line 36 and we are actually using that instead of the function here. I guess this is a merge issue and we can just remove it.

Reply:
Yeah it got added in a merge I guess and I thought it was just something in development that would get used (hence the `allow(dead_code)`)